NEWS.md

```diff
@@ -1,7 +1,20 @@
 ## 0.5.2 to 0.5.3 (unreleased)
 
-(TODO)
-
+ * Events: Subqueries can be extended by the configuration, for example:
```

Comment:
The configuration example should go the example configuration file in server.py (if it is self explaining).